dataverse-webapp/src/main/webapp/dataverse_template.xhtml

```diff
@@ -117,6 +117,13 @@
                 bind_bsui_components();
             });
 
+            var originalPrimeFacesAjaxUtilsSend = PrimeFaces.ajax.Request.send;
+            PrimeFaces.ajax.Request.send = function(cfg) {
+                if (!cfg.oncomplete) {
+                    cfg.oncomplete = bind_bsui_components();
+                }
```

Comment:
This probably won't work for tabs because you override function only if it isn't currently defined.
So if there is already oncomplete handler on button then nothing will happen.
Probably we want something different.
`cfg.oncomplete` should always run `bind_bsui_components()` and if there was some handler before then execute it too